kustomize/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenShift configuration with Kustomize
+
+Resource configuration and secret consumption for OpenShift projects are managed
+using [`kustomize`](https://kubectl.docs.kubernetes.io/guides/introduction/kustomize/).
+
+## Setup
+
+1. Populate `service/secrets/*`.
```

Comment:
Should this PR include kustomize/service/secrets/ folder?

Reply:
Currently you have to put a secrets directory in each of the overlay directories. I'm trying to figure out if this is the best practice of if there's something better.

For instance dev expects a file called

`overlays/dev/secrets/configmap-dev.properties`

and test looks for

`overlays/test/secrets/configmap-test.properties`

It seems like it might be easier if we had like

```
overlays/secrets/configmap-test.properties
overlays/secrets/configmap-dev.properties
```

But perhaps it's fine if it's local to each overlay.

Reply:
This documentation in the READMEI believe is actually incorrect and something I needed to look at and review. I was waiting for Sam to talk about it to see how he ran the ROHQ with their config.

Reply:
These folders are on Dropbox now. The instructors could probably be improved once someone else looks at it but these folders are excluded from being included locally.